common/build/eslint-config-fluid/package.json

```diff
@@ -25,8 +26,7 @@
 		"print-config:recommended": "eslint --config ./recommended.js --print-config ./src/file.ts > ./printed-configs/recommended.json",
 		"print-config:strict": "eslint --config ./strict.js --print-config ./src/file.ts > ./printed-configs/strict.json",
 		"print-config:test": "eslint --config ./index.js --print-config ./src/test/file.ts > ./printed-configs/test.json",
-		"test": "mocha src/test",
-		"test:mocha": "mocha --ignore \"dist/test/types/*\" --recursive dist/test -r node_modules/@fluidframework/mocha-test-setup"
```

Comment:
@WayneFerrao it looks like this isn't used, and it didn't work when I tried running it. Presumably this was a leftover from while you were configuring mocha?